### PR TITLE
missing methods in the pki namespace

### PR DESCRIPTION
**Pull request by ehallander9591**
Added missing methods to pki namespace.

Had to utilize the privateKeyInfoToPem function defined in pki.js in node-forge, and wanted my IDE to acknowledge its existence.

Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Test the change in your own code. (Compile and run.)
- [x ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

I am not sure if I have followed all the conventions. I haven't ever created more than a single pull request for any project, and this one seemed fairly easy.

I did NOT resolve the entire pki namespace, nor did I resolve any other namespace.

I only added what I needed, and if that is selfish and undesired, then I pre-apologize.

### CODE REVIEW

File: types/node-forge/index.d.ts [hunk 1]
```diff
@@ -43,7 +43,9 @@ declare module "node-forge" {
             privateKey: Key;
         }
 
+		function pemToDer(pem: PEM): util.ByteStringBuffer;
         function privateKeyToPem(key: Key, maxline?: number): PEM;
+		function privateKeyInfoToPem(key: Key, maxline?: number): PEM;
```
Comment (westy92) [inline types/node-forge/index.d.ts]: Relevant documentation: https://github.com/digitalbazaar/forge/blob/57ebd54a01b4c38399229a7e7cccbd9a9e73f969/lib/pki.js#L95

File: types/node-forge/index.d.ts [hunk 2]
```diff
@@ -43,7 +43,9 @@ declare module "node-forge" {
             privateKey: Key;
         }
 
+		function pemToDer(pem: PEM): util.ByteStringBuffer;
```
Comment (westy92) [inline types/node-forge/index.d.ts]: Relevant documentation: https://github.com/digitalbazaar/forge/blob/57ebd54a01b4c38399229a7e7cccbd9a9e73f969/lib/pki.js#L36

**Review by westy92 — APPROVED** [review]
Great, thank you!

File: types/node-forge/index.d.ts (hunk 2 above)
Comment (westy92) [inline types/node-forge/index.d.ts]: Indent using spaces instead of tabs.

File: types/node-forge/index.d.ts (hunk 1 above)
Comment (westy92) [inline types/node-forge/index.d.ts]: Indent using spaces instead of tabs.

**Review by westy92 — CHANGES_REQUESTED** [review]
Indent using spaces instead of tabs.

File: types/node-forge/index.d.ts (hunk 1 above)
Comment (ehallander9591) [reply types/node-forge/index.d.ts]: Sorry, I wasn't paying attention after submitting this. I haven't submitted very many PRs before, so it looks like I need to update and move the the state of this PR back to reviewer

**Comment by ehallander9591** [comment]
I just read the last comment from the bot. I will re-open this request.

**Pull request closed without merging**